framework/core/src/Mail/SmtpDriver.php

```diff
@@ -12,11 +12,16 @@
 use Flarum\Settings\SettingsRepositoryInterface;
 use Illuminate\Contracts\Validation\Factory;
 use Illuminate\Support\MessageBag;
-use Swift_SmtpTransport;
-use Swift_Transport;
+use Symfony\Component\Mailer\Transport\Dsn;
+use Symfony\Component\Mailer\Transport\Smtp\EsmtpTransportFactory;
+use Symfony\Component\Mailer\Transport\TransportInterface;
 
 class SmtpDriver implements DriverInterface
 {
+    public function __construct(protected EsmtpTransportFactory $factory)
+    {
+    }
+
```

Comment:
Let's stick to one single format for constructors that declare properties 👍🏼 it makes it easier to differantiate right away if the constructor directly declares props or not.

```suggestion
    public function __construct(
        protected EsmtpTransportFactory $factory
    ) {
    }

```